architecture/topics/router_environment_variables.adoc

```diff
@@ -81,6 +81,11 @@ endif::[]
 |`ROUTER_CIPHERS` | intermediate  | Specify the set of xref:ciphers[ciphers] supported by bind.
 |===
 
+[NOTE]
+====
+Some of the timeouts that we expose interact casuing the total timeout to be longer then specified.
```

Comment:
Should we say:

e.g. ROUTER_SLOWLORIS_HTTP_KEEPALIVE  adjusts "timeout http-keep-alive" , but haproxy will waits on the "tcp-request inspect-delay" which we set to 5s.